### ISSUE TITLE
Washer off produces warnings

### ISSUE THREAD

**Issue by CuriousGerman** [QuickSander/homebridge-mieleathome#45]
A deactivated washer produces warnings in the log caused by invalid temperatue value.

Miele WWV980

[12/07/2021, 07:58:21] [homebridge-mieleathome] This plugin generated a warning from the characteristic 'Current Temperature': characteristic was supplied illegal value: number -327.66 exceeded minimum of 0. See https://git.io/JtMGR for more info.

**Comment by QuickSander** [comment]
Thanks. Will take this along in the next release

**Comment by CuriousGerman** [comment]
Just updated to version 3.3.1 plugin, but still there is the warning.
Homebridge 1.3.4
Node.js: 14.17.1
Nmp: 7.17.0
At Homebridge start it appears twice, then 10 minutes later and then unpredictable.
Quote:
[26/07/2021, 00:18:02] [Miele@home] Discovered device: id: 000xxxxxxxxxxx name: Clementine , model: WWV980
[26/07/2021, 00:18:02] [Miele@home] Re-creating existing accessory with data from cache: Clementine 
[26/07/2021, 00:18:02] [Miele@home] Clementine : Adding temperature sensor.
[26/07/2021, 00:18:03] [Miele@home] Clementine : Initial connection with Miele event server successfully established.
[26/07/2021, 00:18:03] [Miele@home] Clementine : Updating characteristic CurrentTemperature to -327.66.
[26/07/2021, 00:18:03] [homebridge-mieleathome] This plugin generated a warning from the characteristic 'Current Temperature': characteristic was supplied illegal value: number -327.66 exceeded minimum of 0. See https://git.io/JtMGR for more info.
[26/07/2021, 00:18:03] [homebridge-mieleathome] This plugin generated a warning from the characteristic 'Current Temperature': characteristic was supplied illegal value: number -327.66 exceeded minimum of 0. See https://git.io/JtMGR for more info.

**Comment by QuickSander** [comment]
Yes, that is understandable. I did not change anything there in the release I created yesterday.
